bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -152,6 +161,16 @@ spec:
                       by platform services.
                     type: string
                 type: object
+              rhdh:
```

Comment:
Same as above

Reply:
@masayag is it OK to remove this? I know it's being used for RC releases, but we don't use those for now.

Reply:
Agreed not to make changes to the spec in this PR.